UI/Controls/ListenerEditControl.axaml.cs

```diff
@@ -20,6 +23,7 @@ internal partial class ListenerEditControl : UserControl
     public ListenerEditControl()
     {
         InitializeComponent();
+        _autoCompleteBox = this.FindControl<AutoCompleteBox>("TopicAutoCompleteBox");
```

Comment:
search controls in constructor is usefull when we want to subscribe to control event manually in code. And unsubscribe late, maybe

Reply:
Done